src/main/html/index.html

```diff
@@ -24,8 +24,9 @@
   <!-- enabling this will enable oauth2 implicit scope support -->
   <script src='lib/swagger-oauth.js' type='text/javascript'></script>
 
+  <script src='//maxcdn.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.min.js' type='text/javascript'></script>
   <script src='lib/bootstrap.min.js' type='text/javascript'></script>

```

Comment:
@jkho/ @Karin-Basis  two copies of boostrap not creating any problems?

Reply:
No problem as for as I am aware. I didn't even noticed the 2 copies. Was there before i worked on it.

Reply:
Didn't seem to cause any problems for me either.